hs/src/Reach/Simulator/Core.hs

```diff
@@ -260,12 +260,20 @@ data DLVal
   | V_Object (M.Map SLVar DLVal)
   | V_Data SLVar DLVal
   | V_Struct [(SLVar, DLVal)]
+  | V_Maybe DLMaybeVal
```

Comment:
I don't get this and am inclined to disapprove. If it is there because of Haskell, then you want a `Maybe`, if it is there because of Reach, then you want a `V_Data`

Reply:
okay!